test/serverTest.js

```diff
@@ -7,51 +7,50 @@
 'use strict';
 
 const assert = require('chai').assert;
-const jsonfile = require('jsonfile');
 const async = require('async');
 const config = require('../config.json');
 const request = require('request').defaults({'proxy': config.proxy});
 
-var fold = require('../src/backend/fold.js');
-var generator = require('../src/backend/generator.js');
-var dist = require('../src/backend/dist.js');
-var app = require('../src/app');
+let fold = require('../src/backend/fold_v2.js');
+let generator = require('../src/backend/generator.js');
+let dist = require('../src/backend/dist.js');
+let app = require('../src/app');
+let fs = require('fs');
 
 
-
-var data = {
-  event: {endpoint: 'https://raw.githubusercontent.com/fossasia/open-event/master/sample/FOSSASIA16/event'},
-  speakers: {endpoint: 'https://raw.githubusercontent.com/fossasia/open-event/master/sample/FOSSASIA16/speakers'},
-  sessions: {endpoint: 'https://raw.githubusercontent.com/fossasia/open-event/master/sample/FOSSASIA16/sessions'},
-  sponsors: {endpoint: 'https://raw.githubusercontent.com/fossasia/open-event/master/sample/FOSSASIA16/sponsors'},
-  tracks: {endpoint: 'https://raw.githubusercontent.com/fossasia/open-event/master/sample/FOSSASIA16/tracks'},
-  microlocations: {endpoint: 'https://raw.githubusercontent.com/fossasia/open-event/master/sample/FOSSASIA16/microlocations'}
+let data = {
+  event: {endpoint: 'https://raw.githubusercontent.com/fossasia/open-event/master/sample/FOSSASIASummit2016/event'},
+  speakers: {endpoint: 'https://raw.githubusercontent.com/fossasia/open-event/master/sample/FOSSASIASummit2016/speakers'},
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)